sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -437,10 +437,10 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 
 <!-- LINKS -->
 
-[source_code]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/textanalytics/azure-ai-textanalytics
+[source_code]: azure/ai/textanalytics
```

Comment:
before it was pointing to `azure-ai-textanalytics` . Current change seems fine too, but wanted to check it was intentional

Reply:
It was intentional, thanks for checking :)